FastBank.Infrastructure/Repository/EmployeeRepository.cs

```diff
@@ -0,0 +1,59 @@
+﻿using FastBank.Domain;
+using FastBank.Domain.RepositoryInterfaces;
+using FastBank.Infrastructure.Context;
+using FastBank.Infrastructure.DTOs;
+
+namespace FastBank.Infrastructure.Repository
+{
+    public class EmployeeRepository : IEmployeeRepository
+    {
+        private readonly IRepository _repo;
+
+        public const int EmployeesPerPage = 3;
```

Comment:
Constants are written differently. First, constant if it is strongly specific for the class, should be private not public.
Second, constants are commonly written like this: EMPLOYEES_PER_PAGE = 3

Reply:
![image](https://github.com/bulgarchik/FastBank/assets/41870483/f3b9a839-2b5c-4fe6-94a0-ec33b8e16c8d)
Can't change this constant to private use it in another method